testing/browser-functional/browser-tests-build-ci.yml

```diff
@@ -27,6 +27,40 @@ steps:
   displayName: 'Create DateTimeTag for Resource Group'
   # Get-Date on Azure DevOps returns a datetime relative to UTC-0, so "Z" is being used instead of the dynamic "K".
 
+- powershell: |
```

Comment:
Consider using the [PowerShell@2](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/powershell?view=azure-devops) task going forward that is more portable